mautic/mautic
# [UI] Improvements for notes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

fixing the messy notes interface that's been bugging users for months. larger text, better organization, and cleaner design make it easier to read and manage notes. 

Specifically:
- Increases the font size
- Reorders the sequence of information
- Fixes layout bugs
- Changes icons to better express each note category
- Removes a lot of custom code

frequent note-takers will enjoy a more intuitive experience. these changes should boost productivity and reduce eye strain for everyone who relies on notes daily. a small update with big impact on usability.

It seems like we had a bug the last time a PR was merged, some classes in utilities didn't join the app.css file




| Before                                 | After
| -------------------------------------- | ---
|  ![image](https://github.com/mautic/mautic/assets/116097999/128bb762-b627-464c-ac42-b2c344ce8bff)                                      | ![image](https://github.com/user-attachments/assets/75d1c307-5028-4569-9cdd-cfb862f06e62)




---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Contacts
3. Create new
4. Save & Close
5. Go to the Notes tab
6. Add a note for each category
7. See
